UnitTesting-master/src/test/java/parser/JsonParserTest.java

```diff
@@ -0,0 +1,75 @@
+package parser;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.testng.Assert;
+import shop.Cart;
+import parser.JsonParser;
+import parser.Parser;
+import shop.RealItem;
+import shop.VirtualItem;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JsonParserTest {
+    Parser parser;
+    Cart testCart;
+
+    @BeforeEach
+    void setUp() {
+      parser = new JsonParser();
+    }
+
+    @AfterEach
+    void tearDown() {
+    }
+
+    @Test
+    @Tag("positive")
+    void writeToFile_newFileCreated_test() {
+        testCart = new Cart("test-cart");
+        parser.writeToFile(testCart);
+
+        Path filePath = Paths.get("src/main/resources/" + testCart.getCartName() + ".json");
+
+        assertTrue(Files.exists(filePath));
```

Comment:
I recommend to add message, which will be shown on failure, otherwise you will just see "Expected true, but found false"